zplugin-dds/Cargo.toml

```diff
@@ -28,11 +30,15 @@ description = "Zenoh plugin for ROS2 and DDS in general"
 name = "zplugin_dds"
 crate-type = ["cdylib", "rlib"]
 
+[features]
+no_mangle = ["zenoh-plugin-trait/no_mangle"]
+default = ["no_mangle"]
 
 [dependencies]
-zenoh =  { git = "https://github.com/eclipse-zenoh/zenoh"}
-zenoh-ext =  { git = "https://github.com/eclipse-zenoh/zenoh"}
-zenoh-plugin-rest =  { git = "https://github.com/eclipse-zenoh/zenoh"}
+zenoh = { git = "https://github.com/eclipse-zenoh/zenoh" }
+zenoh-ext = { git = "https://github.com/eclipse-zenoh/zenoh" }
+zenoh-plugin-rest = { git = "https://github.com/eclipse-zenoh/zenoh", default-features = false }
```

Comment:
The zplugin-dds lib shouldn't depend on REST plugin.
Only the standalone bin does.